### DISCUSSION TITLE
VISA[NationalInstruments.Visa]  Connot load in Avalonia  but  work in WPF

### DISCUSSION THREAD

**Discussion by ziziran** [AvaloniaUI/Avalonia#15210] [Q&A]
add reference  NationalInstruments.Visa.dll can work in wpf  but in avalonia, system.IO.FileNotFoundException: could not load file or assenbly 'NationalInstruments.Visa'

**Comment by thevortexcloud** [comment]
Have you checked the assembly is in your build output?
